tests/test_plugin.py

```diff
@@ -4141,6 +4141,7 @@ def test_important_plugin_shutdown(node_factory):
     l1.rpc.plugin_start(os.path.join(os.getcwd(), 'plugins/pay'))
 
 
+@pytest.mark.skip(reason="Temporarily disabled expose secrets test")
 @unittest.skipIf(VALGRIND, "It does not play well with prompt and key derivation.")
```

Comment:
Changelog-Removed: `hsmtool` support for mnemonics in non-english languages removed.